platform_plugin_forum_email_notifier/tasks.py

```diff
@@ -72,24 +80,31 @@ def send_email_notification(
     course = get_course_overview_or_none(course_id)
 
     language_preference = get_user_preference(user, LANGUAGE_KEY)
+    post_id = thread_id if title else discussion.get("id")
 
-    send_forum_email_notification(
-        recipient=Recipient(user.id, user.email),
-        language=language_preference,
-        user_context={
+    site = Site.objects.get_current()
```

Comment:
Does this works in the context of a Celery task?

Reply:
I'm not sure about this, because in local the celery tasks run in the LMS pod. Can you test it with the following setting in the LMS: `CELERY_ALWAYS_EAGER = False`.

Of course, we can test in a remote environment and verify it's working

Reply:
Done